src/operator/convolution.cu

```diff
@@ -33,17 +33,21 @@ Operator* CreateOp<gpu>(ConvolutionParam param, int dtype,
   // perform the convolution calculation in 16-bit when the tensor type is
   // also 16-bit.  For NVIDIA architectures earlier than Pascal (so Maxwell
   // and Kepler), the computation precision is always at least 32-bits.
-#if MSHADOW_USE_PASCAL == 1
+
   // true fp16
   int desired_forward_compute_type = dtype;
   int desired_backward_compute_type = dtype;
-#else
-  // pseudo fp16
-  int desired_forward_compute_type =
-    (dtype == mshadow::kFloat16) ? mshadow::kFloat32 : dtype;
-  int desired_backward_compute_type =
-    (dtype == mshadow::kFloat16) ? mshadow::kFloat32 : dtype;
-#endif  // MSHADOW_USE_PASCAL == 1
+
+
+#if CUDNN_MAJOR >= 6
```

Comment:
@DickJC123 i have some question. i not see "fall back to pseudo" code with  MSHADOW_USE_PASCAL=1.the `CuDNNConvolutionOp<DType>::Supports` not check device.